config/environments/profile.rb

```diff
@@ -0,0 +1,41 @@
+# an environment configured for performance profiling.
```

Comment:
now that i have it set up i'm not going to worry about showing errors, etc. new version is  just prod but with inline job queue.